sass/_config.scss

```diff
@@ -45,21 +90,88 @@ $font-path: '../fonts/' !default;
 /// @prop {list} <key>.stack -
 ///   A list of fallback fonts
 ///   for browsers that can't load the given webfont.
-/// @prop {string} <key>.<variant> -
-///   Add keys for each font-variant
-///   (normal, bold, italic, bold italic, etc):
-///   - local path values will have `$font-path` prepended,
-///     and `$font-formats` appended for loading.
-/// @example scss -
+/// @prop {list} <key>.formats -
+///   A list of font formats
+///   (e.g. `'woff2' 'eot'`)
+///   to use when interpolating font-file paths
+///   for font-face imported fonts.
+/// @prop {string | map} <key>.<variant> -
+///   When describing locally-hosted or data-embedded fonts,
+///   provide a key for each font-variant to import.
+///   Multi-word variants should be written as a list
+///   (`'bold' 'italic'`),
+///   rather than a single string
+///   (`'bold italic'`).
+///
+///   - In the simplest use-case,
+///     this can be a string:
+///     a generic path to the appropriate font-files
+///     (`'path/to/my-font-bold'`) with file-extensions removed.
+///     We'll add extensions for you,
+///     based on the `formats` setting.
+///
+///   - For more control over individual variants and formats,
+///     we also accept a map value.
+///     See the `<key>.<variant>.<format>` property below…
+/// @prop {list} <key>.<variant>.path -
+///   An optional string defining the font-path
+///   to interpolate for each given `formats`.
+///   This is the same as providing a string variant-path.
+/// @prop {list} <key>.<variant>.local -
+///   A list of names (strings) to use for `local()` font-src values.
+/// @prop {list} <key>.<variant>.svgid -
+///   Fragment identifier for svg fonts,
+///   to be appended on relevant import URLs.
+/// @prop {list} <key>.<variant>.<format> -
+///   Optionally provide individual font-path overrides,
+///   or base64 data-URIs for specific formats.
+/// @example scss - Defining a font for import
 ///   $fonts: (
 ///     'sans': (
-///       'name': 'FranklinGothic',
+///       'name': 'Museo Sans',
 ///       'stack': ('Helvetica', 'Arial', 'sans-serif'),
-///       'regular': 'franklingothic-regular-filename',
-///       'bold': 'franklingothic-bold-filename',
-///       'bold' 'italic': 'franklingothic-bolditalic-filename',
+///       'formats': ('woff2', 'woff', 'ttf'),
+///       'regular': 'museosans-regular-filename',
```

Comment:
Given "No longer accepts `regular` as a font-variant" above, should this be `'normal'`?

Reply:
In this specific condition (data-eot), or across the board?